profile/README.md

```diff
@@ -1,8 +1,8 @@
 # Welcome to the Lab 🔬
 
-<center>
+<p align="center">
 ![image](https://github.com/instructlab/.github/blob/main/assets/logo.png)
```

Comment:
You have to convert this to `<img .../>`

Reply:
Ah yep!